Add explanation page for box models (#583)
maicos-devel/maicos: docs/src/concepts-explanations/box-models.rst

.. _box-models:

===========================================
Box models for interfacial and slab systems
===========================================

> Many quantities computed by MAICoS — density, dielectric permittivity, velocity — vary

Reviewer: ```suggestion
Many quantities computed by MAICoS — density, dielectric permittivity or velocity profiles — vary
```

> smoothly across an interface. For thermodynamic and continuum-mechanical bookkeeping it

Reviewer: ```suggestion
drastically and in complex way across an interface. For thermodynamic and continuum-mechanical descriptions it
```

> is often convenient to compress such a profile into a piecewise-constant *box model*:

Reviewer: ```suggestion
is often convenient to express such a profile in terms of a piecewise-constant *box model*:
```

> a constant bulk value on each side of an idealized, infinitely thin dividing surface.

Reviewer: ```suggestion
a constant value is assigned on both sides of an idealized, infinitely thin dividing surface.
```

> The position of that surface is not unique; it depends on which quantity is being

Reviewer: ```suggestion
The position of that surface is not unique; it is usually chosen such that the effective response corresonds to the one in bulk on one of the sides, i.e. in general it depends on which quantity is being
```

> conserved across the discontinuity. This page collects the conventions MAICoS users

Reviewer: ```suggestion
conserved across the system. This page collects the typical use-cases for such effective models which MAICoS users
```

> conserved across the discontinuity. This page collects the conventions MAICoS users

Reviewer: I would start the new paragraph here.

> most commonly encounter and shows how to obtain each of them from the analysis output.

Reviewer: ```suggestion
encounter and explains how to obtain each of them from the analysis output.
```
